docs/pages/enterprise/workflow/ssh-approval-slack.mdx

```diff
@@ -28,7 +32,7 @@ This guide will talk through how to setup Teleport with Slack. Teleport to Slack
 
 This guide assumes that you have:
 
-- A running Teleport Cluster
+- Teleport Enterprise 4.2.8 or newer
```

Comment:
The newer plugin versions require Teleport 6.1 due to API changes.

Also as it says above that access requests are now supported in both OSS and Enterprise we can probably drop the Enterprise here?

```suggestion
- Teleport 6.1.0 or newer (or Teleport Cloud)
```